support/serial.c

```diff
@@ -269,11 +319,9 @@ setup_baud_rate (int baud_rate, gboolean *custom_baud_rate)
 	case 75: 
 	    baud_rate = B75;
 	    break;
-	case 50:
 #ifdef B50
+	case 50:
```

Comment:
@ThomasKuehne  This line seems contradictory with your #21377

Reply:
Sorry about that.

After #21377 I look in detail into #20826 and noticed how custom baud rates are handled.

In cases where B50 is undefined - I'm unaware of any current system where that might happen - #21377 is wrong because it returns -1 and thus there is no chance to set 50 via the custom baud rate process.

I'll chancel #21377.